### DISCUSSION TITLE
can I only use lagged regressors (covariates) with autoregressive (AR) disabled?

### DISCUSSION THREAD

**Discussion by JOU1** [ourownstory/neural_prophet#1226] [Q&A - get help using NeuralProphet] [answered]
hi, I'm trying to only use lagged external variables and disable the input of lagged "y" itself. is there a way to do that?
if I set n_lags=0, then both lagged regressor and AR are disabled.
Thanks for any help!!

**Comment by JOU1** [comment, marked as answer]
I resolved it by adding a seperate n_lags to .add_lagged_regressor().

**Reply by judussoari** [reply]
With our new release 0.6.0 (coming in the next couple of days), we will enable separate, and clear definition of lags for AR and covariates. Stay tuned!
